Firebase/Core/FIROptions.m

```diff
@@ -42,8 +42,8 @@
 // Library version ID.
 NSString *const kFIRLibraryVersionID =
     @"5"     // Major version (one or more digits)
-    @"00"    // Minor version (exactly 2 digits)
-    @"01"    // Build number (exactly 2 digits)
+    @"01"    // Minor version (exactly 2 digits)
```

Comment:
This should be the Core version, not Firebase

Reply:
Whoops great catch, thanks.